Looks like `shellcheck` complained about not quoting a variable. Sorry--it's pretty strict but obviously an easy fix.

```
09:30:32 
09:30:32 In rootfs/bin/normalize_storage line 6:
09:30:32   echo $MINION_BUCKET_NAME > /app/objectstore/minio/builder-bucket
09:30:32        ^-- SC2086: Double quote to prevent globbing and word splitting.
09:30:32 
09:30:32 Makefile:56: recipe for target 'test-style' failed
```

--- a/rootfs/bin/normalize_storage
+++ b/rootfs/bin/normalize_storage
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
+  MINIO_BUCKET_NAME=${MINIO_BUCKET:-git}
   mkdir -p /app/objectstore/minio/
-  echo "git" > /app/objectstore/minio/builder-bucket
+  echo "$MINIO_BUCKET_NAME" > /app/objectstore/minio/builder-bucket
   export BUCKET_FILE=/app/objectstore/minio/builder-bucket
 elif [ "$BUILDER_STORAGE" == "azure" ] || [ "$BUILDER_STORAGE" == "swift" ] ; then
   export CONTAINER_FILE=/var/run/secrets/deis/objectstore/creds/builder-container

